src/main/java/org/gelecekbilimde/scienceplatform/auth/model/response/RoleSelfApplicationsResponse.java

```diff
@@ -0,0 +1,23 @@
+package org.gelecekbilimde.scienceplatform.auth.model.response;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+import org.gelecekbilimde.scienceplatform.auth.model.enums.RoleApplicationStatus;
+
+@Getter
+@Setter
+@Builder
+public class RoleSelfApplicationsResponse {
+
+	private String id;
+	private Role role;
+	private RoleApplicationStatus status;
```

Comment:
@Coderabbitai We use UUIDs as `String` throughout the application. If we want to make changes to the IDs in the future, we have made them flexible so that the application is not affected in general. For this reason, don't suggest it again in the next suggestions.